remediation/fence-agents-remediation/Limitations.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+title: Limitations
+parent: Fence Agents Remediation
+grand_parent: Remediation
+nav_order: 3
+---
+
+## Limitations
+
+1. FAR is limited to running FA with only one [action](https://github.com/ClusterLabs/fence-agents/blob/main/doc/FenceAgentAPI.md#agent-operations-and-return-values), reboot.
```

Comment:
- It is preferable to use the full name "Fence Agents" first before using an acronym. It's kind of annoying to do in each page, but you'll never know which first page a reader will face (maybe they just arrived here from some external link).
- Remediation time can be long, not slow.